pkgs/top-level/python-packages.nix

```diff
@@ -147,6 +147,8 @@ in {
 
   alerta-server = callPackage ../development/python-modules/alerta-server { };
 
+  amdgpu-fan = callPackage ../tools/video/amdgpu-fan { };
```

Comment:
generally you'll want to add this to `pkgs/development/python-modules/amdgpu-fan` if you want to keep this as an importable module.

Then in `top-level/all-packages.nix`, you'll want to add:
```
  amdgpu-fan = with python3Packages; toPythonApplication amdgpu-fan;
```

Then use the top-level alias whenever you want to use the application shipped with the package. The toPythonApplication is important because the package wont leak python module information, so you can use many interpreters in the same shell

Reply:
thanks for the feedback @jonringer , did not know about the `python-modules` section and `toPythonApplication`, awesome.

Reply:
so do I understand correctly that you suggest to:
1. move the package from `pkgs/tools/video/amdgpu-fan/default.nix ` `pkgs/development/python-modules/amdgpu-fan`.
2. change the call in `pkgs/top-level/all-packages.nix` appropriately.

Reply:
correct to both